scripts/limpar-prefixo-interno-fonte-dados.py

```
@@ -0,0 +1,164 @@
+    try:
+        with urllib.request.urlopen(req, timeout=90) as r:
+            raw = r.read()
+            return r.status, (json.loads(raw) if raw else None)
+    except urllib.error.HTTPError as e:
+        return e.code, e.read().decode()[:300]
+
+
+def paginar(path: str):
+    out, off = [], 0
+    while True:
+        st, d = rest(f"{path}&limit=1000&offset={off}")
```

Ruff (0.16.1)

[warning] Unpacked variable `st` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)